### PR TITLE
Add Tuple Detection

### PR DESCRIPTION
**Pull request by ShivangiReja**
* Added a list of operation groups for tuple resources in the config file to figure out the tuple resources.
* Added unit tests

Work item - https://dev.azure.com/azure-mgmt-ex/DotNET%20Management%20SDK/_workitems/edit/5857

### CODE REVIEW

**Comment by chamons** [comment]
No changes in shared code, only touches mgmt code.

File: test/TestServerProjects/url/Generated/Configuration.json [hunk 1]
```diff
@@ -17,5 +17,6 @@
   "CredentialHeaderName": "api-key",
   "OperationGroupToResourceType": {},
   "OperationGroupToResource": {},
-  "OperationGroupToParent": {}
+  "OperationGroupToParent": {},
+  "OperationGroupIsTuple": []
```
Comment (pakrym) [inline test/TestServerProjects/url/Generated/Configuration.json:21]: We may want to establish a pattern of default configuration values so we don't serialize empty dictionaries or arrays.

Comment (chamons) [reply test/TestServerProjects/url/Generated/Configuration.json:21]: Maybe issue it for later, I agree though, our config files are getting bloated.

Comment (pakrym) [reply test/TestServerProjects/url/Generated/Configuration.json:21]: Yeah, not as part of this PR.

Comment (ShivangiReja) [reply test/TestServerProjects/url/Generated/Configuration.json:21]: Issue created [here](https://github.com/Azure/autorest.csharp/issues/1246) to track this item.

**Review by m-nash — APPROVED** [review]

**Pull request merged**